integration/tests/karpenter/karpenter_test.go

```diff
@@ -45,11 +45,11 @@ var _ = Describe("(Integration) Karpenter", func() {
 	})
 
 	AfterEach(func() {
-		cmd := params.EksctlDeleteCmd.WithArgs(
-			"cluster", clusterName,
-			"--verbose", "4",
-		)
-		Expect(cmd).To(RunSuccessfully())
+		// cmd := params.EksctlDeleteCmd.WithArgs(
+		//	"cluster", clusterName,
+		//	"--verbose", "4",
+		// )
+		// Expect(cmd).To(RunSuccessfully())
```

Comment:
Let's remove this instead of commenting it. We can revert the changes after we're done with debugging.